### PR TITLE
Action filter events

### PR DESCRIPTION
**Pull request by vpereverzev**

### CODE REVIEW

File: telemetry/actioneventobserver.cpp [hunk 1]
```diff
@@ -89,10 +95,15 @@ bool ActionEventObserver::eventFilter(QObject *watched, QEvent *event)
...
-            Ms::Shortcut* shortcut = Ms::Shortcut::getShortcutByKeySequence(shortCutEvent->key());
+            Ms::Shortcut* shortcut = Ms::Shortcut::getShortcutByKeySequence(shortCutEvent->key(), m_scoreState);
 
             telemetryService()->sendEvent("shortcut", shortcut->key());
             }
 
       return false;
       }
+
+void ActionEventObserver::setScoreState(const Ms::ScoreState state)
```
Comment (Howard-C) [inline telemetry/actioneventobserver.cpp]: Function name comment ;-)

File: telemetry/actioneventobserver.cpp [hunk 2]
```diff
@@ -89,10 +95,15 @@ bool ActionEventObserver::eventFilter(QObject *watched, QEvent *event)
...
+      {
+      m_scoreState = state;
+      }
```
Comment (Howard-C) [inline telemetry/actioneventobserver.cpp]: I think the definition of `setScoreState()` can be moved to the header file?

File: mscore/musescore.cpp [hunk 3]
```diff
@@ -2266,21 +2266,30 @@ void MuseScore::aboutMusicXML()
...
+                  QVariantMap pathMap = actionData.toMap();
+
+                  MasterScore* score = readScore(pathMap.value("filePath").toString());
                   if (score) {
                         setCurrentScoreView(appendScore(score));
                         addRecentScore(score);
                         writeSessionFile(false);
                         }
-                  }
+                  break;
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: No default case?

File: telemetry/actioneventobserver.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply telemetry/actioneventobserver.cpp]: may not work for a 'slot'

**Pull request merged**